ChangeLog

```diff
@@ -35,6 +35,8 @@ Changes
      produced by some optimizing compilers, showing up as failures in
      e.g. RSA or ECC signature operations. Reported in #1722, fix suggested
      by Aurelien Jarno and submitted by Jeffrey Martin.
+   * Re-enable certain ssl-opt.sh tests that depend on a more recent GnuTLS
```

Comment:
Suggestion: Re-enable certain interoperability tests in ssl-opt.sh which had previously been disabled for lack of a sufficiently recent version of GnuTLS on the CI.